src/plugin.js

```diff
@@ -73,7 +74,12 @@ export default {
     } else {
       $ld = initialize({ clientSideId, user, ldOptions, readyBeforeIdentify });
     }
-    // eslint-disable-next-line no-param-reassign
-    Vue.prototype.$ld = Vue.observable($ld);
+
+    if (isVue2) {
+      // eslint-disable-next-line no-param-reassign
+      vue.prototype.$ld = vue.observable($ld);
```

Comment:
By the docs, it seems `reactive` should point to observable or ref depending on the environment; is there a reason to use both?

Reply:
Whoops, yeah there isn't, will fix that